californium-core/pom.xml

```diff
@@ -37,6 +37,35 @@
 	<build>
 		<plugins>
 			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<extensions>true</extensions>
+				<configuration>
+					<instructions>
+						<Export-Package>
+							org.eclipse.californium.core,
```

Comment:
I have taken a look at httpclient-osgi's content. It seems like they are using the same approach as we had taken for californium-osgi, i.e. they inline all content from httpclient and add some OSGi specific classes which introduce a dependency on the OSGi artifacts :-( So, it seems like we cannot depend on these "*-osgi" artifacts without introducing dependencies on OSGi itself ...